game/scripts/npc/abilities/sohei_flurry_of_blows_oaa.txt

```diff
@@ -68,12 +69,6 @@
 				"var_type"					"FIELD_FLOAT"
 				"attack_interval"			"0.6"
 			}
-      // the bonus must be set on sohei_talents_oaa.txt to update its tooltips
-      "05"
-			{
-				"var_type"					      "FIELD_FLOAT"
-				"talent_bonus_radius"			"200"
```

Comment:
As per the comments, this is needed for tooltips. The key should end in `_tooltip` so that's more apparent. The rest seems legit :+1:

Reply:
`"talent_bonus_radius"` is NOT used in description of the ability and the value `"200"` is stored in `sohei_talents_oaa.txt` which can be retrieved using `caster:FindTalentValue()`